cool.py

```diff
@@ -299,9 +299,23 @@ def load_pytorch_model():
     """
     model_weight_path = Path(__file__).parent / "tower_scout" / "xl_250_best.pt"
     yolov5_path = Path(__file__).parent / "yolov5"
-    model = torch.hub.load(str(yolov5_path), "custom", path=str(model_weight_path), source="local")
-    #: if statements (check that .pt file exists, folders, etc.)
 
+    if not model_weight_path.is_file():
+        logging.warning("the model weights file does not exist: %s", model_weight_path)
+
+        return None
```

Comment:
**issue**: these need to throw and stop everything instead of returning `None`.

If the following variable will be `None` it will throw later when you try to read the `conf` property when logging. That will be confusing since the stack trace will not be where the actual problem is.

```py
towerscout_model = _get_model()
```

Reply:
Yep, good point.